black_jack_project/main.py

```diff
@@ -0,0 +1,80 @@
+from replit import clear
+from art import logo
+import random
+
+
+def deal_card():
+  """Returns a random card from the deck."""
+  hand = [11, 2, 3, 4, 5, 6, 7, 8, 9, 10, 10, 10, 10]
+  card = random.choice(hand)
+  return card
+
+def calculate_score(hand):
+  """Take a list of cards and return the score calculated from the cards"""
+
+  if sum(hand) == 21 and len(hand) == 2:
+    return 0
+
+  if 11 in hand and sum(hand) > 21:
+    hand.remove(11)
+    hand.append(1)
+  return sum(hand)
+
+def compare(user_score, computer_score):
+  if user_score > 21 and computer_score > 21:
```

Comment:
This shouldn't be contingent on the computer also going over. Does that make sense? You also should probably include this in the same if-elif-elif-else block as the ones below, as best practice.

Reply:
Okay Grant